src/common/cd-special-req-component.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+<div>
+  <a v-show="!this.hasSpecialReq" class="cd-special-req-component_add" @click="showSpecialReq"> <span><i class="fa fa-plus-circle" aria-hidden="true"></i></span> <span>{{ $t('Add special requirements') }}</span></a>
+  <label v-show="this.hasSpecialReq">{{ $t('Special requirements') }}</label>
```

Comment:
You can wrap the label and the text-area to avoid 2 v-show and make clear the "states" in the template